java/Nikiforov-Oleksandr/src/com/zoolatech/lecture3/tasks/_4/PdfReporter.java

```diff
@@ -0,0 +1,19 @@
+package com.zoolatech.lecture3.tasks._4;
+
+public class PdfReporter extends Reporter {
+    private final String format = " in PDF format";
```

Comment:
since `format` doesn't change, it can be defined as a constant: private static final String FORMAT = ...